jdi-light/src/main/java/com/epam/jdi/light/elements/composite/WebPage2.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.jdi.light.elements.composite;
+
+import com.epam.jdi.light.common.JDIAction;
+
+/**
+ * Created by Roman Iovlev on 25.03.2018 Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class WebPage2 {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'WebPage2' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)